spec/requests/api/v2/people_controller_update_spec.rb

```diff
@@ -0,0 +1,189 @@
+                type: 'ethnicities',
+              },
+            },
+            gender: {
+              data: {
+                id: gender_id,
+                type: 'genders',
+              },
+            },
+          },
+        },
+      }
+    end
+
+    let(:expected_data) do
+      {
+        type: 'people',
+        attributes: {
+          first_names: 'Bob',
+          last_name: 'Roberts',
+          date_of_birth: '1980-01-01',
+          prison_number: 'G3239GV',
+          criminal_records_office: 'CRO0111d',
+          police_national_computer: 'AB/1234567',
+          gender_additional_information: 'info about Bob',
+        },
+        relationships: {
+          gender: {
+            data: {
+              type: 'genders',
+              id: gender_id,
+            },
+          },
+          ethnicity: {
+            data: {
+              type: 'ethnicities',
+              id: ethnicity_id,
+            },
+          },
+        },
+      }
+    end
+
+    let(:expected_included) do
+      []
+    end
+
+    context 'with valid params' do
+      before { put "/api/v2/people/#{person.id}", params: person_params, headers: headers, as: :json }
+
+      it_behaves_like 'an endpoint that responds with success 200'
+    end
+
+    it 'returns the correct data' do
+      put "/api/v2/people/#{person.id}", params: person_params, headers: headers, as: :json
+
+      expect(response_json).to include_json(data: expected_data.merge(id: person.id))
+    end
+
+    describe 'include query param' do
+      before do
+        put "/api/v2/people/#{person.id}#{query_params}", params: person_params, headers: headers, as: :json
+      end
+
+      context 'when including multiple relationships' do
+        let(:query_params) { '?include=gender,ethnicity,profiles' }
+
+        it 'includes the correct relationships' do
+          expect(response_json['included'].count).to eq(3)
+          expect(response_json['included']).to include_json([{ type: 'ethnicities' }, { type: 'genders' }, { type: 'profiles' }])
+        end
+      end
+
+      context 'when does NOT include any relationship' do
+        let(:query_params) { '' }
+
+        it 'does NOT include any relationships' do
+          expect(response_json).not_to include('included')
+        end
+      end
+
+      context 'when including a non existing relationship' do
+        let(:query_params) { '?include=gender,non-existent-relationship' }
+
+        it 'responds with error 400' do
+          response_error = response_json['errors'].first
+
+          expect(response_error['title']).to eq('Bad request')
+          expect(response_error['detail']).to include('["non-existent-relationship"] is not supported.')
+        end
+      end
+    end
+
+    describe 'webhook and email notifications' do
+      before do
+        allow(Notifier).to receive(:prepare_notifications)
+        put "/api/v1/people/#{person.id}", params: person_params, headers: headers, as: :json
```

Comment:
this is going to the wrong URL - should be /v2/, not /v1/

Reply:
😱 thanks to spot that!